### ISSUE TITLE
No output file?

### ISSUE THREAD

**Issue by desmodus1984** [lbcb-sci/raven#37]
Hi, 
I ran raven with the following commands and I got nothing out of it.
#!/bin/bash
#SBATCH --time=120:00:00
#SBATCH --mem=170G
#SBATCH --ntasks=48
#SBATCH --job-name=Raven_ONTq-set1-1K
#SBATCH --account=PHS0338
module load python && source activate assembly-Y
raven ONTq-set1-1K.fa -t 48

Is there a way to get the output as a file - Fasta?

There are just a file called raven.cereal, and that's it.

I don't even know the final assemble size or anything... to check for completeness or else.

**Comment by rvaser** [comment]
Hello,
the assembly is sent to stdout, so you have to redirect it to a file with `> asm.fa`. You do not have to rerun it from the beginning, just submit a resume run with `raven ONTq-set1-1K.fa -t 48 --resume > asm.fa`.

Best regards,
Robert

**Comment by ardy20** [comment]
Hi Robert
Thanks for this. It would be great if you add this into the main page.

**Comment by rvaser** [comment]
Hi,
this information is written in the usage section of the README and when you run `raven --help`.

Best regards,
Robert

**Comment by wanyuac** [comment]
> Hi,
> this information is written in the usage section of the README and when you run `raven --help`.
> 
> Best regards,
> Robert

Sorry, but I could not find such information in the usage section or the output of `raven --help` (v2.8.1-b1676) as of today. In fact, I had the same issue just now, until I figured it out that raven writes outputs to stdin.

**Comment by rvaser** [comment]
Raven's latest version is 1.5.0, not sure what is with the version in brackets. If you run `raven --help` you get the following (I added an arrow to emphasize the output to stdout part):
```bash
usage: raven [options ...] <sequences>

  # default output is to stdout in FASTA format  <----------------------------------
  <sequences>
    input file in FASTA/FASTQ format (can be compressed with gzip)

  options:
    --weaken
      use larger (k, w) when assembling highly accurate sequences
    -p, --polishing-rounds <int>
      default: 2
      number of times racon is invoked
    -m, --match <int>
      default: 3
      score for matching bases
    -n, --mismatch <int>
      default: -5
      score for mismatching bases
    -g, --gap <int>
      default: -4
      gap penalty (must be negative)
    --graphical-fragment-assembly <string>
      prints the assembly graph in GFA format
    --resume
      resume previous run from last checkpoint
    --disable-checkpoints
      disable checkpoint file creation
    -t, --threads <int>
      default: 1
      number of threads
    --version
      prints the version number
    -h, --help
      prints the usage
```

**Comment by jpinus** [comment]
I used the "--resume" command, because "raven input.fa --resume > output.fa" always just generated a FASTA-file which was empty. But since today I experience an error when using the "--resume"-command:
$ raven unmapped.fastq.gz --resume > test.fa
[raven::] loaded previous run 4.904889s
[raven::] loaded 2223978 sequences 216.895768s
Illegal instruction

I also tried it with data that worked before but I get the same error. 

any idea what can cause this?
